### DISCUSSION TITLE
Labels per entity

### DISCUSSION THREAD

**Discussion by jrodan** [kalkih/mini-graph-card#1042] [Q&A] [answered]
Hi, quick question: Is there a way to show the temperature next to each graph like the labels on the left bottom?
I want to see with one view all temperatures + labels + graph data.
thanks!
![Bildschirmfoto 2023-12-07 um 14 22 38](https://github.com/kalkih/mini-graph-card/assets/6048704/8189fc36-d897-4d18-a2b0-ada41e555d60)
PS: if the name of the entity would be visible on the top right next to the temperature, this would also be enough for now.

**Comment by ildar170975** [comment]
Post a picture of what you mean.

**Reply by jrodan** [reply]
![Bildschirmfoto 2023-12-11 um 12 08 58](https://github.com/kalkih/mini-graph-card/assets/6048704/a4bd2035-34fa-47ce-86ba-97782993622d)

Variant 1.: just add the name to the temperature
Variant 2.: add the temperature to the graph directly 
Variant 3.: Show the name + temperature in 2-3 columns on top

Cool would be in Variant 1 e.g. that the name matches the color of the graph without having a duplicate over the main graph area.

Is there any way to achieve that?

Thanks!

**Reply by ildar170975** [reply, marked as answer]
> Is there any way to achieve that?

No way currently - you cannot achieve this by supported options.
Everything needs changes in the code (except var.1 - could be achieved by card-mod).

> Variant 2.: add the temperature to the graph directly

Not nice.
Curves of different graphs may be placed close to each other; displaying states near curves may cause overlaying.

> Variant 1.: just add the name to the temperature

See open issues - there is a similar FR with proposed solutions (at east this may be achieved by card-mod right now w/o any changes in the card).
